packages/mixin-cache-as-bitmap/src/index.ts

```diff
@@ -247,6 +282,15 @@ DisplayObject.prototype._initCachedDisplayObject = function _initCachedDisplayOb
         resolution: this.cacheAsBitmapResolution || renderer.resolution,
     });
 
+    if (this.cacheAsBitmapMultisample !== undefined)
+    {
+        renderTexture.framebuffer.multisample = this.cacheAsBitmapMultisample;
+    }
+    else if (renderer.gl.getContextAttributes().antialias)
+    {
+        renderTexture.framebuffer.multisample = MSAA_QUALITY.HIGH;
+    }
```

Comment:
I this this is good, but it is a change in the default behavior and should be considered a breaking change. We can either remove it or provide a static property to opt-in to this behavior, then make it the default in v7. Thoughts @dev7355608?
 
```suggestion
```

Reply:
I removed it. I think we don't really need an opt-in property. For example, anyone who wants to force multisampling everywhere could simply change the default like this.
```js
DisplayObject.prototype._cacheAsBitmapMultisample = PIXI.MSAA_QUALITY.HIGH;
```